tests/yadism/cf/test_cfh_g1.py

```diff
@@ -0,0 +1,151 @@
+import numpy as np
+from eko.thresholds import ThresholdsAtlas
+
+from yadism import observable_name as on
+from yadism.coefficient_functions.heavy import f2_nc as h_f2_nc
+from yadism.coefficient_functions.heavy import f3_nc as h_f3_nc
+from yadism.coefficient_functions.heavy import g1_nc as h_g1_nc
+
+
+# todo: compare results to Daniel de Florian figure 4-6
```

Comment:
if you want to keep this check, also consider the extras folder ... I guess you think of 2210.12014? they only consider light there, I believe (my thesis is not involved) - still you may want to check them